config/locales/help_text/fr.yml

```diff
@@ -8,7 +8,7 @@ fr:
       birthdate: Date de naissance
       email: Adresse courriel
       full_name: Nom complet
-      intro_html: Il s'agit de la seule information que %{app_name} partagera avec
-        %{sp}
+      intro_html: 'Il s''agit de la seule information que %{app_name} partagera avec
```

Comment:
Looks like you added an extra single quote in `s''agit`.

Reply:
woops. good catch

Reply:
@monfresh hmm this appears to be added by the normalizer? Is that escaping the single quote

Reply:
Yeah, this is how you escape a single quote in a single quoted string in YAML